raml-parser-interface/src/test/resources/api-vendor-utils-test-resources/swagger.yaml

```diff
@@ -0,0 +1,66 @@
+swagger: "2.0"
```

Comment:
I would add some comments at the top of this API as well

Reply:
done